package.json

```diff
@@ -1,15 +1,15 @@
 {
-  "name": "poll",
+  "name": "poll-with-routes",
   "author": "Ryland Goldstein",
-  "version": "0.1.1",
+  "version": "0.2.1",
   "private": true,
   "license": "MIT",
   "dependencies": {
     "@reshuffle/code-transform": "0.0.3",
     "@reshuffle/db": "^0.5.3",
-    "@reshuffle/fetch-runtime": "0.0.5",
-    "@reshuffle/local-proxy": "^0.2.12",
-    "@reshuffle/server-function": "0.0.6",
+    "@reshuffle/fetch-runtime": "0.0.6",
+    "@reshuffle/local-proxy": "^0.2.14",
```

Comment:
This is the result of `npm install`. No carets added to 0.0.x version since semver considers them incompatible with 0.0.y versions. All will be caret when we bump to 1.0.0.

Reply:
Thanks @vogre, you're amazing.